packages/instrumentation-mysql2/test/mysql.test.ts

```diff
@@ -36,6 +36,8 @@ import {
 import * as assert from 'assert';
 import { MySQL2Instrumentation, MySQL2InstrumentationConfig } from '../src';
 
+process.env.OTEL_SEMCONV_STABILITY_OPT_IN = 'http/dup,database/dup';
+
```

Comment:
Would it be possible to add test cases for non-dup cases ?

Reply:
+1 I think is important, so we know we're not sending extra telemetry the user doesn't want, therefore costing more (resource/money)

An example on how I did for pg:
https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/packages/instrumentation-pg/test/pg-pool.test.ts#L994